### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by Splyth**
Using MIT, I just slapped my name on there. If you want I can throw you guys on there too

### CODE REVIEW

**Comment by Splyth** [comment]
https://github.com/Splyth/slack_bot/issues/54

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright 2020 Scott Mackensen
```
Comment (amiller-sta) [inline LICENSE]: Might want to add MIT license at the top of the file, at least that's how I've seen other repos do it https://github.com/cmderdev/cmder/blob/master/LICENSE#L1
Not sure how Github parses the license but that might be how

**Comment by amiller-sta** [comment]
> Using MIT, I just slapped my name on there. If you want I can throw you guys on there too
Since you've done most of the work, I think it's ultimately your call. I've only added like two features, but others have contributed more than me.

File: LICENSE (hunk 1 above)
Comment (Splyth) [reply LICENSE]: done

**Pull request merged**